addons/mail/static/src/discuss/core/web/discuss_sidebar_categories.js

```diff
@@ -83,8 +83,9 @@ export class DiscussSidebarCategories extends Component {
 
     filteredThreads(category) {
         return category.threads.filter((thread) => {
+            const isChannelMember = thread.type !== "channel" || !thread.readonly;
             return (
-                (thread.is_pinned || thread.group_based_subscription) &&
+                ((thread.is_pinned && isChannelMember) || thread.eq(this.store.discuss.thread)) &&
```

Comment:
I think ZEL fixed the pin/member part separately.

Reply:
@zel-odoo Do you confirrm ? Can I safely remove that change from my pull request ?